Fix tests and update PathwayMapper

msalihaltun opened the pull request:
Fix e2e tests for toast in PatientView, which was replaced with a message banner. Check for existence of banner instead. Remove a condition in a test that was waiting for toast to disappear, since that functionality was replaced with a message banner witha 'close' button. Resolve https://github.com/cBioPortal/cbioportal-frontend/pull/3458#issuecomment-742709284.

Update PathwayMapper to latest version where the PatientView help modal is better spaced out (removed breaklines). See https://github.com/iVis-at-Bilkent/pathway-mapper/commit/0e087998b4c97e1bf6a3887632b005e5f3d95e13.

Remove messageBanner property from PathwayMapperComponent of PatientViewPathwayMapper since it has become an optional property.

msalihaltun committed:
Fix tests and update PathwayMapper

  Fix e2e tests for toast in PatientView, which was replaced with
  a message banner. Check for existence of banner instead. Remove
  a condition in a test that was waiting for toast to disappear, since
  that functionality was replaced with a message banner with
  a 'close' button.

  Update PathwayMapper to latest version where the PatientView help
  modal is better spaced out (removed breaklines). See
  https://github.com/iVis-at-Bilkent/pathway-mapper/commit/0e087998b4c97e1bf6a3887632b005e5f3d95e13

  Remove messageBanner property from PathwayMapperComponent of
  PatientViewPathwayMapper since it has become an optional property.

onursumer commented on end-to-end-test/remote/specs/core/screenshot.spec.js:
```diff
@@ -165,8 +165,7 @@ function runResultsTestSuite(prefix, options = {}) {
         browser.waitForVisible('#cy', 10000);
-        browser.waitForExist('.Toastify__toast', 4000);
-        browser.waitUntil(() => !$('.Toastify__toast').isExisting());
+        browser.waitForExist('#pathway-mapper-message-box', 4000);
```
We should use `data-test` prop here:

```suggestion
        browser.waitForExist('div[data-test="pathwayMapperMessageBox"]', 4000);
```

onursumer commented on src/shared/lib/pathwayMapper/PathwayMapperMessageBox.tsx:
```diff
@@ -12,6 +12,7 @@ const PathwayMapperMessageBox: React.FunctionComponent<PathwayMapperMessageBoxPr
     const isWarningMessage = message !== props.loadingMessage;
     return (
         <div
+            id="pathway-mapper-message-box"
```
We should use `data-test` prop here:

```suggestion
            data-test="pathwayMapperMessageBox"
```

onursumer reviewed (changes requested):
We should use `data-test` prop for test purposes

msalihaltun committed:
Update tests and PathwayMapper version

  PathwayMapper commit: https://github.com/iVis-at-Bilkent/pathway-mapper/commit/6875fd9ee1f4a24efefbc19baabf5a0c4d58ea42

  Use 'data-test' prop instead of 'id' for e2e test.
